packages/conventional-commit-lint/src/app.ts

```diff
@@ -18,4 +18,9 @@ import { GCFBootstrapper } from 'gcf-utils';
 import appFn from './conventional-commit-lint';
 
 const bootstrap = new GCFBootstrapper();
-module.exports.conventional_commit_lint = bootstrap.gcf(appFn);
+async function run() {
+    const opts = await bootstrap.getProbotConfig();
+    console.info(opts.cert);
```

Comment:
we can probably revert this bit, will be curious to see if the underlying bug crops up again.